modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/change_lists/view_conflicts.jsp

```diff
@@ -33,7 +33,9 @@ portletDisplay.setShowBackIcon(true);
 portletDisplay.setURLBack(backURL);
 %>
 
-<div class="container-fluid container-fluid-max-xl container-form-lg">
+<clay:container
+	className="c-py-3 c-py-md-5"
```

Comment:
This is now the 3rd variation of replacement of `container-form-lg` after
- [c-pt-mb-5](https://github.com/eduardoallegrini/liferay-portal/pull/70/files#diff-8dfe0f21bf041bdc6a670f09cacb2b8cR37)
- [c-pb-md-5](https://github.com/eduardoallegrini/liferay-portal/pull/70/files#diff-6cd4d3076d850004b9515213ff54cc16R27)

Makes me doubt the direction as it seems very hard to achieve consistency. It might well happen that we had consistency and now we don't want it...

Reply:
the correct one is `c-py-3 c-py-md-5` but in one of the cases we have a menubar with negative margin https://github.com/liferay/clay/blob/0273c22d1f6a9dfbd22e39f9490d7a9da05e53b7/packages/clay-css/src/scss/components/_grid.scss#L48 , so in order to keep the same space I change it to `c-pb-mb-5` the other one must be a typo